test/puppetlabs/puppetdb/http/metrics_test.clj

```diff
@@ -37,22 +33,23 @@
   [req]
   (assoc-in req [:headers "accept"] "text/plain"))
 
-(deftestseq metrics-set-handler
+(let [app (server/build-app)]
+  (deftestseq metrics-set-handler
```

Comment:
I actually had it like that in the original incarnation but decided against it for two reasons:

1. It was verbose, a function and a dynamic variable declaration for use by a single test, with no plans to add more functionality to the app or add more tests, it just seemed unnecessary.

2. We're planning on abstracting the metrics application to its own repo or project(?) so that it can be shared by different applications. So 1. seemed even more applicable.

Totally don't mind changing it, an easy one, but that was my reasoning.